### PR TITLE
Fix: Issue #222(changed Twitter link)

### PR DESCRIPTION
**Pull request by S-125**
## Purpose

The purpose of this Pull Request is to fix #<issue-number>

### Preview Link

https://pr-{PR_NUMBER}-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by S-125** [comment]
Please add hacktoberfest-accepted label to my PR before merging it.

File: dark/index.html [hunk 1]
```diff
@@ -147,7 +147,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Twitter</span>
-        <a href="https://twitter.com/Yoshitha2008" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>
+        <a href=" https://x.com/Yoshitha_R" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:150]: Remove the space here @S-125 
```suggestion
        <a href="https://x.com/Yoshitha_R" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by YoshithaRathnayake** [comment]
> Please add hacktoberfest-accepted label to my PR before merging it.

Ah sure! So if he approve the PR according to the guidelines there are no problems you're hacktoberfest PR is accepted but I am doing the both steps so don't worry @S-125 And I am going to make a new hacktoberfest project

https://github.com/YoshithaRathnayake/Hacktoberfest-Project

SO keep in touch (Starred the rep)

**Review by YoshithaRathnayake — COMMENTED** [review]
Changes are fine!. Make sure you have completed these tasks! @S-125 

- [ ] Starred this repository

- [ ] Followed me on social media (Visit my GitHub profile for more info)

After doing these things I'll merge this for sure

**Comment by S-125** [comment]
@YoshithaRathnayake  I have done both

**Comment by YoshithaRathnayake** [comment]
> @YoshithaRathnayake I have done both

Atleast follow me in github 😂🤌🏻

**Comment by S-125** [comment]
I am so sorry I followed u on insta and forgot about github

**Comment by YoshithaRathnayake** [comment]
> I am so sorry I followed u on insta and forgot about github

Ah nice ♥ @S-125

**Review by YoshithaRathnayake — APPROVED** [review]
Good job! Thank you @S-125

**Comment by YoshithaRathnayake** [comment]
So I am going to do a another hacktoberfest project soon so, if you couldn't complete the 4 tasks go keep in touch. @S-125 

https://github.com/YoshithaRathnayake/Hacktoberfest-Project

**Pull request merged**